Lombiq.Tests.UI.Samples/Tests/FrontendTests.cs

```diff
@@ -0,0 +1,46 @@
+using Lombiq.Tests.UI.Extensions;
+using OpenQA.Selenium;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Lombiq.Tests.UI.Samples.Tests;
+
+public class FrontendTests : FrontendUITestBase
+{
+    public FrontendTests(ITestOutputHelper testOutputHelper)
+        : base(testOutputHelper)
+    {
+    }
+
+    // The interesting details were in FrontendUITestBase, here we just show that you can freely interact with the pages
+    // served by frontend server the same way as usual. In this case we have an HTTP file server, so you can navigate
+    // these directories and files. If we had a large client application that interacts with the headless OC in the
+    // backend, we would be able to do something more interesting but that's outside the scope of this demo.
+    [Fact]
+    public Task FrontendServerShouldStartWithTest() =>
+        ExecuteFrontendTestAfterSetupAsync(
+            async context =>
+            {
+                // Don't forget that if you want to interact with the frontend manually, you can just switch the context
+                // back to the back end and use the interactive mode extension method.
```

Comment:
I don't understand the question.

Reply:
I believe that this should be https://github.com/Lombiq/UI-Testing-Toolbox/pull/421/files/bdb345aa3710e59b52ce149c161b1eff6fa36e1e#r1846935116, otherwise I don't understand.

Reply:
Nope. The point is that you can explore the frontend in interactive mode, but you can't just call `await context.SwitchToInteractiveAsync();`. You have to switch the UI test context scope back to the backend first (`context.SwitchToBackend();`) or it won't work, because the `await context.SwitchToInteractiveAsync();` will try to visit the page on the frontend's URL.

Reply:
I see. Can't `SwitchToInteractiveAsync` detect where it's called from and change the URL accordingly? Having to remember this is exposing an implementation particularity. Or at least an overload that makes this easier to discover, something along the lines of `SwitchToInteractiveModeFromFrontendWithSwitchToBackendAsync` (this is terribly long, but you get the idea).

Reply:
It's not just the SwitchToInteractiveAsync, but every method that uses Lombiq.Tests.UI.Shortcuts (because those controller actions are in the backend). I don't think it's viable nor worth the extra complexity.